modules/kmm-creating-module-cr.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
```

Comment:
This module is also not formatted as a procedure. However, the reconciliation loop is not a user procedure anyway - would suggest retagging this as a concept module and adjusting the title accordingly.

Reply:
Changed content-type to CONCEPT and reworded title to "The Module Custom Resource Definition".